### PR TITLE
changed capitalization in WinformsControlsTest namespace name

### PR DESCRIPTION
**Pull request by Tanya-Solyanik**
* changed capitalization in WinformsControlsTest->WinFormsControlsTest namespace name
* added DesignerCategory("default") attribute to the test forms

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/11505)

### CODE REVIEW

File: src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/ChartControl.cs [hunk 1]
```diff
@@ -4,40 +4,40 @@
 using System.Drawing;
 using System.Windows.Forms.DataVisualization.Charting;
 
-namespace WinformsControlsTest
+namespace WinFormsControlsTest;
+
+[DesignerCategory("default")]
```
Comment (KlausLoeffelmann) [inline src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/ChartControl.cs:9]:
```suggestion
[DesignerCategory("Default")]
```

Can you do a quick check, if we use the captial 'D'?

File: src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/MainForm.cs [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
 using System.Windows.Forms.IntegrationTests.Common;
 using Microsoft.Win32;
 using WindowsFormsApp1;
-using WinformsControlsTest.UserControls;
+using WinFormsControlsTest.UserControls;
 
-namespace WinformsControlsTest;
+namespace WinFormsControlsTest;
 
 [DesignerCategory("code")]
```
Comment (KlausLoeffelmann) [inline src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/MainForm.cs:13]: Should we capitalize for consistency?

**Review by KlausLoeffelmann — APPROVED** [review]
:shipit:

File: src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/MainForm.cs (hunk 2 above)
Comment (Tanya-Solyanik) [reply src/System.Windows.Forms/tests/IntegrationTests/WinformsControlsTest/MainForm.cs:13]: I'd rather fix this capitalization in the next PR

**Comment by Olina-Zhang** [comment]
We can see this change in main branch of the cloned winforms repo:
image

**Pull request merged**